tests/unit/components/ArtefactMenuItem-test.js

```diff
@@ -0,0 +1,30 @@
+"use strict"
+
+import { mount } from '@test'
+import ArtefactMenuItem from '~/components/ArtefactMenuItem.vue'
```

Comment:
I can take a look at these "click" troubles at some point and see if I can figure it out. Might be late Saturday before I can do that though...

Reply:
No rush, I just got a bottle of rum and am ready for the weekend.

Reply:
Nice! 💯 ! (more of a whiskey guy myself but I have a developing appreciation for rum ...)